invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,3 +29,24 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task(ignore_result=True)
+def update_expired_embargos():
+    """Release expired embargoes."""
+    service = current_rdm_records.records_service
+    embargoed_q = f"access.embargo.active:true AND access.embargo.until:" \
+                  f"{{* TO {arrow.utcnow().datetime.strftime('%Y-%m-%d')}}}"
+    # Retrieve overdue embargoed records
+    restricted_records = service.scan(identity=system_identity, q=embargoed_q)
+    for restricted_record in restricted_records.to_dict()["hits"]["hits"]:
```

Comment:
I think this can be changed to:

```python
for restricted_record in restricted_records.hits:
```

and then below do `_id=restricted_record.id`

Reply:
Yes, we can use `for restricted_record in restricted_records.hits:` but not `restricted_record.id` since `restricted_record` it's a dict. Thanks!